src/components/Error.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  errorMessage: string;
+  setErrorMessage: (errorMessage: string) => void;
+}
+
+export const Error: React.FC<Props> = ({ errorMessage, setErrorMessage }) => {
```

Comment:
```suggestion
export const ErrorNotification: React.FC<Props> = ({ errorMessage, setErrorMessage }) => {
```
It's better to name it as a notification to avoid name collision with throwing Errors